src/test/java/ru/training/at/hw2/ex2/JdiPageSeleniumSecondTest.java

```diff
@@ -0,0 +1,149 @@
+package ru.training.at.hw2.ex2;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+
+public class JdiPageSeleniumSecondTest {
+    private WebDriverWait webDriverWait;
+    WebDriver webDriver;
+    WebElement webElement;
+
+    final String testPageUrl = "https://jdi-testing.github.io/jdi-light/index.html";
+    final String userName = "Roman";
+    final String password = "Jdi1234";
+    final String diffElementsPageUrl = "https://jdi-testing.github.io/jdi-light/different-elements.html";
+
+    @BeforeMethod(alwaysRun = true)
+    public void driverSetup() {
+        System.setProperty("webdriver.chrome.driver",
+                "C:\\Drivers\\chromedriver_win32\\chromedriver.exe");
+        webDriver = new ChromeDriver();
+        webDriver.manage().window().maximize();
+        /*webDriver.manage().timeouts()
+                .implicitlyWait(10, TimeUnit.SECONDS);*/
+    }
```

Comment:
Looks like `@BeforeMethod` is the same as for src/test/java/ru/training/at/hw2/ex1/JdiPageSeleniumFirstTest.java. It could be extracted to the base class

Reply:
Extracted to base class